bazel/repository_locations.bzl

```diff
@@ -490,6 +490,18 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-07-10",
         cpe = "cpe:2.3:a:nodejs:node.js:*",
     ),
+    com_github_nodejs_llhttp = dict(
+        project_name = "llhttp",
+        project_desc = "Parser for HTTP messages written in C",
+        project_url = "https://github.com/nodejs/llhttp",
+        version = "5.0.0",
+        sha256 = "1c575cf37c748fd0d67fd8ff9761c71ef28e7c1c3dc336c4e8a7f9787b34e470",
+        strip_prefix = "llhttp-release-v{version}",
+        urls = ["https://github.com/nodejs/llhttp/archive/release/v{version}.tar.gz"],
+        use_category = ["dataplane_core"],
+        release_date = "2021-04-03",
+        cpe = "N/A",
+    ),
```

Comment:
the CI dependencies precheck is failing, please take a look

Reply:
The release date differing from GitHub UI will always fail me.

Reply:
Hey @asraa I'm going to fix this shortly. The current code looks at the UTC for the last commit for the release. This can be different from when the release was actually cut. With my new version you'll still have to enter the UTC for the release but at least you don't have to reverse engineer back through commits /cc @htuch